src/jitlog/build.lua

```diff
@@ -0,0 +1,46 @@
+local stdout = io.stdout
+local args = {...}
+
+if args[1] == "GC64" then
+  GC64 = true

```

Comment:
Shouldn't there be a `GC64=false` branch?

Reply:
nil is false in Lua as well. Think of the way a #ifdef check in C is used

Reply:
Ug, that's another terrible language feature that I don't think we should be relying on. That means that making a variable name typo doesn't cause an error. Let's just add a false branch, and make clear to the user what's going on. [The C preprocessor is good fun, but it's also a complete mess, so I don't want to emulate that!]

Reply:
Its just a common Lua idiom imo

Reply:
I know it's possible in Lua, but we sholdn't ever be using this feature: it's been known for decades to cause problems.

Reply:
I wish a mis-spelled variable caused an error in Lua :)

(scurries back into the burrow from whence he came)

Reply:
code update